### ISSUE TITLE
[Topic] Automated accessibility testing

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#174]
URL: https://wpaccessibility.org/docs/testing/automated/
Code: https://github.com/wpaccessibility/wp-a11y-docs/blob/main/docs/testing/automated/index.md

Please read [How to contribute](https://wpaccessibility.org/docs/contribute/) before you start.

## Tasks

This content should be reviewed , restructured, expanded and maybe divided into multiple pages.
The links need to be checked and updated.

**Comment by FredSchenk** [comment]
The url of the article by Marco Zehe has changes to:
https://www.marcozehe.de/how-to-use-nvda-and-firefox-to-test-your-web-pages-for-accessibility/

He has some dead links to the old url on his own site too.

**Comment by rianrietveld** [comment]
@FredSchenk Thank! The broken link is now fixed 🙏

**Comment by rianrietveld** [comment]
Add Playwright, resource:  [Automating Accessibility Testing with Playwright](https://rowdy.codes/talks/20260508-devworld/).
